docs/_data/nav.json5

```diff
@@ -85,6 +85,7 @@
             { label: '<hx-popover>', path: 'hx-popover' },
             { label: '<hx-progress>', path: 'hx-progress' },
             { label: '<hx-radio-control>', path: 'hx-radio-control' },
+            { label: '<hx-radio-set>', path: 'hx-radio-set' },
```

Comment:
please alphabetize (nothing before something)

Reply:
i think we're doing something before nothing, correct? if you open this file look at lines 66-67, 91-92, etc. @CITguy

Reply:
so, alphabetically `-` (`<hx-radio-set>`) sorts before `>` (`<hx-radio>`)

Reply:
This is because the ASCII value of `-` is 45 and `>` is 62, so `-` is sorted first. In vim, I can sort a selection of lines by using the `:sort` action and it sorts using ASCII value. I think VSCode has a similar capability (unconfirmed).

Reply:
Sorting will be ... sorted... out in another PR, so that we can correct as many cases as possible.